sentences/de/binary_sensor_HassGetState.yaml

```diff
@@ -16,7 +16,7 @@ intents:
           device_class: battery
 
       - sentences:
-          - "(ist|sind) <irgend> <batterie> [<area>] {bs_battery_states:state}"
+          - "(ist|sind) [<irgend>] <batterie> [<area>] {bs_battery_states:state}"
```

Comment:
@mib1185 thanks for the review 👍 yes it's unrelated and I know that this should be avoided and I like to stick to the guidelines. I made these changes anyway because I didn't want to unnecessarily slow down the translation process for the German intents by creating separate PRs for one file which would finally result in a conflict. As these are just little fixes and the tests are passing I hope you can agree on merging them anyway? Shouldn't harm anyone or the project.

Reply:
breaking rules, just to accelerate something is not a good idea 😉 
it still counts to say [We prefer a lot of small contributions over a few large ones. Contributions that contain a lot of changes are hard to review.](https://developers.home-assistant.io/docs/voice/intent-recognition/contributing)

Reply:
Hm I have a different opinion especially after visiting your link 😉 it only says to keep PRs small and limited to 1 language and 1 domain what I did here.

I don't really want to discuss about that because I just want to translate the sentences. But to me the process feels too strict compared to the guidelines (without any visible benefit) and somehow frustrating. Maybe this can be improved in the future.

Reply:
it is more about, keep an PR as small as possible, to make it easy as possible to review. as more changes on different topics are combined into one PR, as much complex is the review ans those might also lasts longer.
Further these changes does also need changes of related tests.

So it still counts "We prefer a lot of small contributions over a few large ones." 😉 
Even they might result in conflicts, but resolving these might be less effort in opposite to review an PR with multiple topics, those multiple dependencies to keep in mind and check.